docs/reference/glossary-of-terms.md

```diff
@@ -0,0 +1,300 @@
+
+#### Relationship
+
+A [Relationship](#Relationship) is a type connection between two ontology elements. The first element is called the [subject](#Subject), and the second one the [Object](#Object), with the type of connection being the [Relationship Type](#Relationship-Type). Sometimes Relationships are equated with [Triples](#Triple) in [RDF](#RDF) but this can be confusing, because some relationships map to _multiple_ triples when following the OWL RDF serialization. An example is the relationship "finger part-of hand", which in OWL is represented using a [Existential Restriction](#Existential-Restriction) that maps to 4 triples.
+
+#### Relationship Type
+
+See [predicate](#predicate)
+
+### ROBOT
+
+A toolkit for transforming and interacting with ontologies. Full documentation can be found at http://robot.obolibrary.org/
+
+#### Semantic Similarity
+
+A means of measuring similarity between either pairs of ontology concepts, or between entities annotated using ontology concepts. There is a wide variety of different methods for calculating semantic similarity, for example [Jaccard Similarity](#Jaccard-Similarity) and [Information Content](#Information-Content) based measures.
+
+#### Semantic SQL
+
+Semantic SQL is a proposed standardized schema for representing any RDF/OWL ontology, plus a set of tools for building a database conforming to this schema from RDF/OWL files. See [Semantic-SQL](https://github.com/INCATools/semantic-sql)
+
+#### SPARQL
+
+The standard query language and protocol for Linked Open Data on the web or for RDF [triplestores](#Triplestore) - used to query ontologies.
+
+#### SSSOM
+
+Simple Standard for Sharing Ontological Mappings (https://github.com/mapping-commons/sssom).
+
+#### Subject
+
+The "left" side of a [Triple](#Triple).
+
+#### Subset
+
+A named collection of elements, typically grouped for some purpose.
+
+#### Term
+
+Usually used to mean [Class](#Class) and [Individuals](#Individual), however sometimes used to refer to wider [OWL entities](#OWL-Entity).
+
+#### Text Annotation
+
+The process of annotating spans of texts within a text document with references to ontology terms, or the result of this process. This is frequently done automatically. The [Bioportal](#Bioportal) implementation provides text annotation services.
+
+#### Triple
+
+A set of three entities that codifies a statement about semantic data in the form of [Subject](#Subject)-[predicate](#predicate)-[Object](#Object) expressions (e.g., "Bob is 35", or "Bob knows John"). Also see [Relationship](#Relationship).
+
+#### Triplestore
+
+A purpose-built database for the storage and retrieval of triples through semantic queries. A triple is a data entity composed of subject–predicate–object, like "Bob is 35" or "Bob knows Fred".
+
+#### Ubergraph
+
+A [Triplestore](#Triplestore) and a [Ontology Repository](#Ontology-Repository) that allows for [SPARQL](#SPARQL) querying of integrated [OBO](#OBO) ontologies.
+
```

Comment:
Kinda understates what Ubergraph is and why it's useful.  Link to original and maybe take some text from their readme.